docs/upgrade/capsule.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: documentation
+title: Capsule Upgrade
+sidebar: sidebars/documentation.html
+---
+
+# Capsule Upgrade
+
+Katello supports upgrades from version 2.0.  For users transitioning from 1.4, please see - [Transition Guide](/docs/installation/2.0-transition.html).
+
+## Step 1 - Operating System
+
+Ensure your operating system is fully up-to-date:
+
+```
+# yum -y update
+```
+
+## Step 2 - Repositories
+
+Update the Foreman and Katello release packages:
+
+  * RHEL6 / CentOS 6:
+
+```
+  # yum update -y http://fedorapeople.org/groups/katello/releases/yum/2.1/katello/RHEL/6Server/x86_64/katello-repos-latest.rpm
```

Comment:
What do you think about not using `#` then a user could double-click highlight the entire line and copy? It could just be me, but I do that a lot.